website/partials/_index-teaser.ejs

```diff
@@ -11,7 +11,7 @@
                 <div style="background:rgba(255, 255, 255, 0.95);margin-top:50px;margin-bottom:50px;padding:35px;border-radius:15px;display:inline-block;max-width:750px;">
                     <div class="row" style="margin-left:0px;margin-right:0px;">
                         <h1 class="header-apache-text">Apache <span class="sp-green">S</span><span
```

Comment:
@dominikriemer we could update the index page similar to the README of the main repository, what do you think?

Reply:
with the new texts and images?

Reply:
yes, for example. Just as an idea :)